src/SummaryRow.tsx

```diff
@@ -1,18 +1,23 @@
 import { memo } from 'react';
+import clsx from 'clsx';
 import { css } from '@linaria/core';
 
 import { cell, rowClassname } from './style';
 import { getColSpan } from './utils';
 import SummaryCell from './SummaryCell';
-import type { RowRendererProps } from './types';
+import type { CalculatedColumn, RowRendererProps } from './types';
+import { useRovingRowRef } from './hooks';
 
 type SharedRowRendererProps<R, SR> = Pick<RowRendererProps<R, SR>, 'viewportColumns' | 'rowIdx'>;
 
-interface SummaryRowProps<R, SR> extends SharedRowRendererProps<R, SR> {
-  'aria-rowindex': number;
+interface SummaryRowProps<R, SR>
+  extends SharedRowRendererProps<R, SR>,
+    Pick<React.HTMLAttributes<HTMLDivElement>, 'aria-rowindex'> {
```

Comment:
`aria-rowindex` is optional now, is this intentional?

Reply:
It was unintentional, reverted it